nixos/modules/misc/meta.nix

```diff
@@ -44,6 +53,19 @@ in
         '';
       };
 
+      wikiPages = mkOption {
+        type = moduleListOf types.str;
+        internal = true;
+        default = [];
+        example = [
+          "https://wiki.nixos.org/wiki/NixOS"
```

Comment:
I'd like some discussion here. This implementation currently allows you to add any sort of links, despite being meant for `wiki.nixos.org`. Is this something we'd like to constrain, either by having people write `NixOS` for `https://wiki.nixos.org/wiki/NixOS` or by asserting the `https://wiki.nixos.org/wiki/` prefix?. Or is the ability to add arbitrary links considered a feature?

Reply:
IMO it’s a good idea to only allow paths to the wiki. Allowing to add arbitrary links would permit to add links to the unofficial wiki or even personal blogs/websites. I don’t think we want that.